open-ce/conda_env_file_generator.py

```diff
@@ -43,7 +43,6 @@ def write_conda_env_file(self,
         It returns a list of paths to the files that were writeen.
```

Comment:
Bit of  a nit, but this still says it returns a list.
```suggestion
        It returns the path to the file that was written.
```